src/user/screens/profile.screen.js

```diff
@@ -144,31 +173,33 @@ class Profile extends Component {
       navigation,
     } = this.props;
     const { refreshing } = this.state;
-    const initialUser = navigation.state.params.user;
     const isPending = isPendingUser || isPendingOrgs;
     const userActions = [
       isFollowing
         ? translate('user.profile.unfollow', language)
         : translate('user.profile.follow', language),
     ];
 
+    const orgs = [];
+
+    if (!user) {
```

Comment:
Instead of doing this, we could draw the screen, display the information we already have (user avatar and name) and draw loading elements for the rest until the data is retrieved from the API.